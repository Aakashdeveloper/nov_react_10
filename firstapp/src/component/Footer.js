import React,{Fragment} from 'react';

function Footer(props){
    return(
        <Fragment>
            <hr/>
            <center>
                <h3>&copy; NareshIT {props.year} {props.month}</h3>
            </center>
        </Fragment>
    )
}


/*const Footer =()=>{
    return(
        <div>
            <hr/>
            <center>
                <h3>&copy; NareshIT</h3>
            </center>
        </div>
    )
}*/


export default Footer;