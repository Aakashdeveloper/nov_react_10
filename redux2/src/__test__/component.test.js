import React from 'react';
import {create} from 'react-test-renderer';
import BaseButton from '../component/Basebutton';
import Footer from '../component/Footer';

describe('Functional test',() => {
    test('Button test',() => {
        const component = create(<BaseButton/>)
        const instance = component.root;
        const button = instance.findByType("button");
        button.props.onClick();
        expect(button.props.children).toBe("Some Thing")
    })
    test('Snapshot testing',() => {
        let tree = create(<Footer/>);
        expect(tree.toJSON()).toMatchSnapshot()
    })
})