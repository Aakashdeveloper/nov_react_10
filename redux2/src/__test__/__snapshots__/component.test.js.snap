// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Functional test Snapshot testing 1`] = `
<footer>
  <div>
    <hr />
    Â© Developer Funnel
  </div>
</footer>
`;
